server/database/config/db_build.sql

```diff
@@ -0,0 +1,67 @@
+BEGIN;
+  DROP TABLE IF EXISTS employees,questions,employees_questions,forms,employee_form,form_questions
+  CASCADE;
+
```

Comment:
Let`s rename ids to contain his table name, id for question table make it question_id and so on

Reply:
I think the correct way to use identifiers as primary keys in their tables is to type as they are, but when we need them as foreign keys, we should write them in relation to their table names.